### PR TITLE
krunvm: add page, add Spanish translation

### PR DESCRIPTION
**Pull request by jparrill**
Adding Krunvm utility in english and spanish languages

This utility is compatible in Linux and OSX.

Signed-off-by: Juan Manuel Parrilla Madrid <jparrill@redhat.com>

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): krunvm 0.2.3**

### CODE REVIEW

**Comment by jparrill** [comment]
Hey @slp it's ok to add krunvm command to tldr app?

**Comment by slp** [comment]
> Hey @slp it's ok to add krunvm command to tldr app?

Sure, thanks!

File: pages/osx/krunvm.md
Comment (EmilySeville7cfg) [inline pages/osx/krunvm.md:8]:
```suggestion
`krunvm create {{docker.io/fedora}} --cpus {{number_of_vcpus}} --mem {{memory_in_mbs}} --name "{{name}}"`
```

- Optional fix: first placeholder can be enclosed to `{{` and `}}` as this positional argument accepts not only `docker.io/fedora` value.
- Mandatory fix: [placeholder format violation](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming): use [snake_case](https://wikipedia.org/wiki/snake_case) for multi-word placeholders.
- Optional fix: last placeholder can be quoted if it can contain any special characters for POSIX-like shells.

File: pages/osx/krunvm.md
Comment (EmilySeville7cfg) [inline pages/osx/krunvm.md:12]:
```suggestion
`krunvm start "{{name}}"`
```

The same issue as above.

File: pages/osx/krunvm.md
Comment (EmilySeville7cfg) [inline pages/osx/krunvm.md:20]:
```suggestion
`krunvm changevm --cpus {{number_of_vcpus}} --mem {{memory_in_mbs}} --name "{{new_vm_name}}" "{{current_vm_name}}"`
```

The same issue as above.

File: pages/osx/krunvm.md
Comment (EmilySeville7cfg) [inline pages/osx/krunvm.md:24]:
```suggestion
`krunvm delete "{{name}}"`
```

- Almost the same issue as above.
- Optional fix: consistency violation: placeholder can be renamed to `{{name}}` to be consistent with the first example (and others).

File: pages/osx/krunvm.md
Comment (EmilySeville7cfg) [inline pages/osx/krunvm.md:10]:
```suggestion
- Start a specific image:
```

- Optional fix: `a specific` may sound better.
- Optional fix: `MicroVM` can be implicitly supposed to be a type of OCI images.

File: pages/osx/krunvm.md
Comment (EmilySeville7cfg) [inline pages/osx/krunvm.md:14]:
```suggestion
- List images:
```

The same issue as above.

File: pages/osx/krunvm.md
Comment (EmilySeville7cfg) [inline pages/osx/krunvm.md:18]:
```suggestion
- Change a specific image:
```

The same issue as above.

File: pages/osx/krunvm.md
Comment (EmilySeville7cfg) [inline pages/osx/krunvm.md:22]:
```suggestion
- Delete a specific image:
```

The same issue as above.

File: pages.es/osx/krunvm.md
Comment (EmilySeville7cfg) [inline pages.es/osx/krunvm.md:1]: Warning: this page contains 1 or more mandatory to fix issues (source: original page).

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Changes requested: 1 of more mandatory fixes found.
- These pages should be reviewed by `kruvm` users.
- Thanks for a contribution! 🚀

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
@jparrill, at least two reviewers should approve PR and nobody else should request changes to make it technically mergeable. Is there any reason [according to our maintainers guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md#ii-handling-prs) why their review requests were canceled?

**Comment by jparrill** [comment]
> 



> @jparrill, at least two reviewers should approve PR and nobody else should request changes to make it technically mergeable. Is there any reason [according to our maintainers guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md#ii-handling-prs) why their review requests were canceled?

I don't think so but, let me know if that happen ;)

File: pages.es/osx/krunvm.md
Comment (kbdharun) [inline pages.es/osx/krunvm.md:4]:
```suggestion
> Más información: <https://github.com/containers/krunvm>.
```

This is what we use for all Spanish pages.

File: pages/osx/krunvm.md
Comment (kbdharun) [inline pages/osx/krunvm.md:12]:
```suggestion
`krunvm start "{{image_name}}"`
```
How about using this?

File: pages/osx/krunvm.md
Comment (kbdharun) [inline pages/osx/krunvm.md:24]:
```suggestion
`krunvm delete "{{image_name}}"`
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr. And Thanks for your contribution. I have some suggestions for this page. 

I am not adding my approval to this page, as the Spanish translation must be reviewed by a native speaker. @navarroaxel can you check the translation.

> Why their review requests were canceled?

@EmilySeville7cfg whenever someone rerequests review in their PR GitHub dismisses other review requests lol.

**Comment by EmilySeville7cfg** [comment]
> @EmilySeville7cfg whenever someone rerequests review in their PR GitHub dismisses other pending
review requests lol.

Interesting feature...

**Comment by jparrill** [comment]
> I am not adding my approval to this page, as the Spanish translation must be reviewed by a native speaker. @navarroaxel can you check the translation.

Perfect, btw I'm native spanish but it's fine to be reviewed by an other one :)

**Comment by jparrill** [comment]
> This page must be placed in `common` as it is compatible with both Linux and Mac os

You got it

**Comment by jparrill** [comment]
@EmilySeville7cfg could you please tag other maintainer to get this merge plz?

File: pages/common/krunvm.md
Comment (navarroaxel) [inline pages/common/krunvm.md:6]: We try to avoid abbreviations. what is a µVM? and which is the difference with a classic VM?

File: pages/common/krunvm.md
Comment (navarroaxel) [inline pages/common/krunvm.md:8]:
```suggestion
`krunvm create {{docker.io/fedora}} --cpus {{number_of_vcpus}} --mem {{memory_in_megabytes}} --name "{{name}}"`
```

File: pages.es/common/krunvm.md
Comment (navarroaxel) [inline pages.es/common/krunvm.md:8]: Please translate the placeholders for get this easier for non-English speakers.
```suggestion
`krunvm create {{docker.io/fedora}} --cpus {{numero_de_vcpus}} --mem {{memoria_en_megabytes}} --name "{{nombre}}"`
```

File: pages.es/common/krunvm.md
Comment (navarroaxel) [inline pages.es/common/krunvm.md:12]:
```suggestion
`krunvm start "{{nombre}}"`
```

**Review by kbdharun — APPROVED** [review]
LGTM after @navarroaxel's suggestions.

File: pages/common/krunvm.md
Comment (jparrill) [reply pages/common/krunvm.md:6]: There is plenty of info about this check this out https://williamdurand.fr/2022/07/11/containers-and-micro-virtual-machines/

File: pages.es/common/krunvm.md
Comment (EmilySeville7cfg) [inline pages.es/common/krunvm.md:8]:
```suggestion
`krunvm create {{docker.io/fedora}} --cpus {{numero_de_vcpus}} --mem {{memoria_en_megabytes}} --name "{{nombre}}"`
```

Mandatory fix (multiple violations in the current page): command syntax is broken: `--nombre` option is unknown to `krunvm`.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/krunvm.md
Comment (navarroaxel) [inline pages/common/krunvm.md:3]:
```suggestion
> CLI-based utility for creating MicroVMs from OCI images.
```

File: pages/common/krunvm.md
Comment (navarroaxel) [reply pages/common/krunvm.md:6]: ok, it's a new "concept" and doesn't have a page in Wikipedia.

File: pages.es/common/krunvm.md
Comment (navarroaxel) [inline pages.es/common/krunvm.md:3]:
```suggestion
> Utilidad basada en CLI para crear micro máquinas virtuales utilizando imagenes OCI.
```

File: pages.es/common/krunvm.md
Comment (navarroaxel) [inline pages.es/common/krunvm.md:6]:
```suggestion
- Crea una micro máquina virtual basada en Fedora:
```

**Review by EmilySeville7cfg — APPROVED** [review]
No unmentioned issues found.

**Comment by jparrill** [comment]
Squashed and ready to be merged

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**
